pom.xml

```diff
@@ -57,6 +57,17 @@
 
   <build>
     <plugins>
+      <!-- Note:  this is needed for some IDEs to detect as a Java project.
```

Comment:
QUESTION: I am guessing this is for Visual Studio Code?

Reply:
Yeah, although I actually had trouble importing into Eclipse too in a new workspace.  I'm sure there was some other plugin I needed for Eclipse, but no matter what I tried I could not get it to pick up this thing as a Java project.  Since the plugin doesn't use Groovy I figured this is harmless and might make other people's lives easier